common/api/endpoint/ApiEndPoint.ts

```diff
@@ -29,6 +29,9 @@ export const ApiEndPoint = {
             'metadata'
           ),
       },
+      CommentableDescription: {
+        one: () => apiPath('assessment', 'sections', 'commentableDescription'),
```

Comment:
I would just use /assessment/data/descriptions 

And introduce Assessment.Data property , we'll refactor table data and data points later.
What do you think ?